contrib/guix/guix-build

```diff
@@ -154,8 +154,8 @@ avail_KiB="$(df -Pk "$VERSION_BASE" | sed 1d | tr -s ' ' | cut -d' ' -f4)"
 total_required_KiB=0
 for host in $HOSTS; do
     case "$host" in
-        *darwin*) required_KiB=440000 ;;
-        *mingw*)  required_KiB=7600000 ;;
+        *darwin*) required_KiB=8388608 ;;
```

Comment:
>  due to a change made in the build process (-static-libgcc).

Can you explain a bit more how this is related to `-static-libgcc`, given you haven't changed the allowance for any Linux builds, and macos/mingw don't use `-static-libgcc`?

Reply:
> I did notice discrepancies for darwin and mingw though, which I've attempted to correct for.

That is fine, but you'll have to properly explain the changes.

https://github.com/kevkevinpal/bitcoin/issues/200 could be an oversight where someone ran the build twice by accident?

Reply:
@maflcko I proposed the changes to the build gates by just running a build and checking before/after snapshots of disk usage. Do we need more detailed info like tracking which files the extra disk space use comes from?